src/Tizen.MachineLearning.Inference/Tizen.MachineLearning.Inference/MlInformation.cs

```diff
@@ -0,0 +1,283 @@
+        /// <since_tizen> 13 </since_tizen>
+        public void SetNumber(string key, int value)
+        {
+            if (string.IsNullOrEmpty(key))
+                throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, "The property key is invalid");
+
+            NNStreamerError ret = NNStreamerError.InvalidParameter;
+
+            switch(_type)
+            {
+                case InfoType.Option:
+                    ret = Interop.Util.SetOptionValue(_handle, Interop.Util.StringToIntPtr(key), Interop.Util.IntToIntPtr(value), null);
+                    break;
+                case InfoType.Information:
+                    ret = NNStreamerError.NotSupported;
+                    Log.Error(NNStreamer.TAG, "InfoType Iniformation does not support set value");
+                    break;
+            }
+
+            NNStreamer.CheckException(ret, "Failed to set string value");
+        }
+
+        private IntPtr GetInformationHandle(string key)
+        {
+            if (string.IsNullOrEmpty(key))
+                throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, "The property key is invalid");
+
+            IntPtr value = IntPtr.Zero;
+            NNStreamerError ret = NNStreamerError.InvalidParameter;
+
+            switch(_type)
+            {
+                case InfoType.Option:
+                    ret = Interop.Util.GetOptionValue(_handle, Interop.Util.StringToIntPtr(key), out value);
+                    break;
+                case InfoType.Information:
+                    ret = Interop.Util.GetInformationValue(_handle, Interop.Util.StringToIntPtr(key), out value);
+                    break;
+            }
+
+            NNStreamer.CheckException(ret, "Failed to get information handle from key");
+
+            return value;
+        }
+
+        /// <summary>
+        /// Gets a string value of key in MlInformation instance.
+        /// </summary>
+        /// <param name="key">The key to be set.</param>
+        /// <returns>The string value of the key.</returns>
+        /// <feature>http://tizen.org/feature/machine_learning.inference</feature>
+        /// <exception cref="ArgumentException">Thrown when the method failed due to an invalid parameter.</exception>
+        /// <since_tizen> 13 </since_tizen>
+        public string GetString(string key)
+        {
+            IntPtr value = GetInformationHandle(key);
+            if (value == IntPtr.Zero)
+                throw NNStreamerExceptionFactory.CreateException(NNStreamerError.InvalidParameter, "Could not find information handle from key");
+
+            return Interop.Util.IntPtrToString(value);
+        }
+
+        /// <summary>
+        /// Gets a int value of key in MlInformation instance.
+        /// </summary>
+        /// <param name="key">The key to be set.</param>
+        /// <returns>The string value of the key.</returns>
```

Comment:
suggestion:
description: The integer value of the key.
method name: GetNumber/SetNumber -> GetInteger / SetInteger ?

Reply:
Updated description and method name :)